### PR TITLE
Stable branch

### PR DESCRIPTION
**Pull request by TimeEngineer**
Hi, I would like to use this crate with stable `rust`

### CODE REVIEW

File: winfsp-sys/src/bindings.rs
Comment (TimeEngineer) [inline winfsp-sys/src/bindings.rs:1]: Is this needed to maintain such file ?

**Comment by chyyran** [comment]
Hi, sorry for the delayed response. My GitHub notifications have not been working lately. 

I will review this within the week.

File: winfsp-sys/src/bindings.rs
Comment (chyyran) [reply winfsp-sys/src/bindings.rs:1]: Yes, for docs.rs purposes.

File: winfsp/src/lib.rs [hunk 1]
```diff
@@ -22,8 +14,8 @@ pub use error::FspError;
 pub use error::Result;
 
 pub use init::{winfsp_init, winfsp_init_or_die, FspInit};
-
 pub use widestring::{U16CStr, U16CString};
+pub use windows::*;
```
Comment (chyyran) [inline winfsp/src/lib.rs:18]: This import doesn't seem needed. If it is, do not glob import `windows`.

File: winfsp/src/vsb.rs [hunk 2]
```diff
@@ -54,6 +54,11 @@ impl<T> VariableSizedBox<T> {
     pub fn len(&self) -> usize {
         self.size
     }
+
+    /// Returns `true` if it contains no elements.
+    pub fn is_empty(&self) -> bool {
```
Comment (chyyran) [inline winfsp/src/vsb.rs:59]: This isn't used anywhere and doesn't make sense for the semantics of `VariableSizedBox`

File: winfsp/src/filesystem/internals/fileinfo.rs [hunk 3]
```diff
@@ -54,7 +54,7 @@ impl OpenFileInfo {
             unsafe {
                 self.normalized_name.write(prefix);
                 self.normalized_name
-                    .map_addr(|addr| addr.wrapping_add(1))
+                    .wrapping_offset(1)
```
Comment (chyyran) [inline winfsp/src/filesystem/internals/fileinfo.rs:57]: Please add a cfg here for strict provenance semantics if available.

**Review by chyyran — CHANGES_REQUESTED** [review]
Most of the nightly features used have been stabilized since this crate was released, which is good.

Please add a cfg out for strict_provenance semantics when compiling under nightly.

**Comment by chyyran** [comment]
Addressed in #8. Thanks for your contribution.

You can disable usage of nightly features like so:

```toml
[dependencies.winfsp]
default-features = false
# notify if you need filesystem notifications, delayload is needed for build-time helpers.
features = ["notify", "delayload"]
```

**Pull request closed without merging**